<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Product</class>
 <widget class="QFrame" name="Product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>204</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>658</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>658</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>658</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,1">
   <item>
    <widget class="QLabel" name="Image">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>150</width>
       <height>150</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../../../../time/import/picture.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,2,4">
     <item row="4" column="1">
      <widget class="QLabel" name="OldPrice">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
         <strikeout>true</strikeout>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:red; text-decoration:line-through;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="Price">
       <property name="text">
        <string>Цена: </string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="NewPrice">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="Type">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="Description">
       <property name="text">
        <string>Описание товара: </string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="Creator">
       <property name="text">
        <string>Производитель: </string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="Seller">
       <property name="text">
        <string>Поставщик: </string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="3">
      <widget class="QLabel" name="Counter">
       <property name="text">
        <string>Единица измерения: </string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="Count">
       <property name="text">
        <string>Количесвто на складе: </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="Discount">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Update">
       <property name="text">
        <string>Изменить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Delete">
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
